packages/dds/tree/src/test/domains/json/jsonCursor.bench.ts

```diff
@@ -121,6 +133,53 @@ function bench(
     }
 }
 
+function jsObjectBench(
+    data: {
+        name: string;
+        getJson: () => any;
+        dataConsumer: (directObj: any, calculate: (...operands: any[]) => void) => any;
+    }[],
+) {
+    for (const { name, getJson, dataConsumer } of data) {
+        const json = getJson();
+
+        benchmark({
+            type: BenchmarkType.Measurement,
+            title: `clone JS Object: '${name}'`,
+            before: () => {
+                const cloned = clone(json);
+                assert.deepEqual(cloned, json, "clone() must return an equivalent tree.");
+                assert.notEqual(cloned, json, "clone() must not return the same tree instance.");
+            },
+            benchmarkFn: () => {
+                clone(json);
+            },
+        });
+
+        benchmark({
+            type: BenchmarkType.Measurement,
+            title: `sum JS Object: '${name}'`,
+            before: () => {
+                // TODO: add checks before running
```

Comment:
I initially had that in since the clone direct test did, but I removed the TODO statement as no checks seem to be needed for `sum` and `averageTwoValues`